<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crusader Kings 2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>launcher/logo.ico</normaloff>launcher/logo.ico</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>699</width>
      <height>402</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(launcher/background.jpg);</string>
    </property>
    <widget class="QPushButton" name="buttonReg">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(launcher/button_small.jpg);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="textDLC">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select DLC's</string>
     </property>
    </widget>
    <widget class="QListWidget" name="modList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="textMod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select mods to play</string>
     </property>
     <property name="buddy">
      <cstring>modList</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRun">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(launcher/button_big.jpg);
border-width: 1px;</string>
     </property>
     <property name="text">
      <string>Start CK2</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="dlcList">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>webView</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>webView</tabstop>
  <tabstop>modList</tabstop>
  <tabstop>buttonRun</tabstop>
  <tabstop>buttonReg</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
